osbs/utils/__init__.py

```diff
@@ -148,7 +148,7 @@ def checkout_git_repo(git_url, target_dir=None, commit=None, retry_times=GIT_MAX
 
 
 def clone_git_repo(git_url, target_dir=None, commit=None, retry_times=GIT_MAX_RETRIES, branch=None,
-                   depth=None):
+                   depth=None, exists_ok=False):
```

Comment:
Shouldn't be needed after the changes in https://github.com/containerbuildsystem/atomic-reactor/pull/1708

Reply:
removed.